src/utils/viewer/viewer.cpp

```diff
@@ -974,12 +974,15 @@ void Viewer::mouseReleaseEvent(MouseEvent& event) {
 }
 
 void Viewer::mouseScrollEvent(MouseScrollEvent& event) {
-  if (!event.offset().y()) {
+  // shift+scroll is forced into x direction on mac, seemingly at OS level, so
+  // use both x and y offsets.
+  float scrollModVal = event.offset().y() + event.offset().x();
```

Comment:
```suggestion
  float scrollModVal = event.offset().max();
```

shorter / less error-prone, maybe? On touchpads you can have both directions and so the two could cancel each other out. I think the `max()` could work reasonably well there too.

Reply:
is it an abs max?  Can be + or -

Reply:
Also, the x displacement is roughly 10x as big as the y, which was why i added them, but as you say, max would work too, if it is a magnitude max.

Reply:
Ah, no it's not, good point. `Mn::Math::abs(event.offset()).max()` then :)

Reply:
How about this : 

```
  float scrollModVal = abs(event.offset().y()) > abs(event.offset().x())
                           ? event.offset().y()
                           : event.offset().x();
```

Reply:
This way then any mouse scrolling on 1d or 2d mice, platform independent, will accomplish something expected, maybe?

Reply:
Yeah let's do that :)